### PR TITLE
s3.1 Additional text to explain sigs on server cert & DC can be different

### PR DESCRIPTION
**Pull request by seanturner**

### CODE REVIEW

**Comment by seanturner** [comment]
Addresses #89.

File: draft-ietf-tls-subcerts.md [hunk 1]
```diff
@@ -275,8 +275,11 @@ mechanisms like proxy certificates {{?RFC3820}} for several reasons:
   which rely on a cryptographic binding between the entire certificate and the
   delegated credential, cannot.
 * Each delegated credential is bound to a specific signature algorithm for use
-  use in the TLS handshake ({{RFC8446}} section 4.2.3).  This prevents
+  in the TLS handshake ({{RFC8446}} section 4.2.3).  This prevents
   them from being used with other, perhaps unintended signature algorithms.
+  This prevents them from being used with other, perhaps unintended signature
```
Comment (grittygrease) [inline draft-ietf-tls-subcerts.md:280]: Repeated sentence

**Review by grittygrease — CHANGES_REQUESTED** [review]
Remove repeated sentence.

File: draft-ietf-tls-subcerts.md [hunk 2]
```diff
@@ -275,8 +275,11 @@ mechanisms like proxy certificates {{?RFC3820}} for several reasons:
...
+  algorithms. The signature algorithm bound to the delegated credential and
+  the one in the servers certificate can be different.
```
Comment (grittygrease) [inline draft-ietf-tls-subcerts.md:282]: This applies to more than just server certificates. Maybe say:
"The signature algorithm bound to the delegated credential can be chosen independently of the set of signature algorithms supported by the end-entity certificate."

**Review by grittygrease — CHANGES_REQUESTED** [review]

**Review by jsalowey — APPROVED** [review]

**Review by chris-wood — APPROVED** [review]

**Pull request merged**